scripts/agt/v1.linux64_conf.txt

```diff
@@ -8,7 +8,7 @@ if [[ $AGENT_REV_LOCAL == "v1.hokkaido.213" ]]; then
 fi
 
 # default if AGENT_REV is present in env
-if [[ $AGENT_REV_LOCAL != "v1.hokkaido"* ]]; then
+if [[ $AGENT_REV_LOCAL != "v1.hokkaido"* && $AGENT_REV_LOCAL != "1.0.0-r"* ]]; then
```

Comment:
Need some explanation.

Reply:
@ayasuda-ge .. Added new condition to handle new semantic format `1.0.0-r.xxx`. If no agent version mentioned in either format will be defaulted to v1.hokkaido.212